### PR TITLE
updated things to 1.4.5, compiles!

### PR DESCRIPTION
**Pull request by hapm**
Did a diff over all files that caused problems when compiling and applied changes accordingly. Added comments from mcp in some files, but not everywhere. Cleaned up some // Spout comments. I hope this helps. I hope so, as this is my first commit to spoutcraft

### CODE REVIEW

File: src/main/java/net/minecraft/src/Entity.java [hunk 1]
```diff
@@ -93,8 +202,7 @@
 	public boolean partiallyInWater = false;
 	public org.spoutcraft.api.entity.Entity spoutEntity;
 	public UUID uniqueId = UUID.randomUUID();
-	public boolean wasOnGround;
-	public boolean clientonly = false;
+	public boolean wasOnGround;	public boolean clientonly = false;
```
Comment (Top-Cat) [inline src/main/java/net/minecraft/src/Entity.java:205]: Did MCP do this? ._.

Happens again below...

Comment (hapm) [inline src/main/java/net/minecraft/src/Entity.java:205]: No, that was a bad merge of mine. Scrumbled it up by using TortoiseMerge and different line endings, sorry for that. I'll come up with a fix for that in an hour or two. Did a recheck of things while merging with lenis0012's request and found this error as well.

**Comment by Zidane** [comment]
Can you squash it down to one commit potentially as well?

**Comment by hapm** [comment]
Sorry for that, i'll do it an rerequest the pull.

**Comment by hapm** [comment]
I should have checked astyles.cfg before i started the merge. Need to redo a bunch of things as the new mcp version overwrote some customizations from SpoutDev. Will take its time...

**Comment by hapm** [comment]
Will need for this until tomorrow...

**Pull request closed without merging**